index.html

```diff
@@ -118,9 +121,87 @@ <h2 class="blog-underline-headline">Latest Blog Posts</h2>
           </p>
         </div>
         <div class="blog-folders">
-          <div class="test"><p class="testone"></p></div>
-          <div></div>
-          <div></div>
+          <div class="first-column">
+            <div class="folder">
+              <div class="folder-headline">
+                <p class="folder-name">Just Another Blog Post</p>
+              </div>
+            </div>
+            <div class="folder-info">
+              <ul class="info-list">
+                <li><img src="src/img/blog/blog-user.png"  alt=""/></li>
+                <li><p>By Author</p></li>
+                <li><img src="src/img/blog/calendar.png"  alt=""/></li>
+                <li><p>9 November 2014</p></li>
+                <li><img src="src/img/blog/blog-comment.png"  alt=""/></li>
+                <li><p>3</p></li>
+              </ul>
+            </div>
+            <div class="folder-paragraph">
+              <p class="folder-words">
+                Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                Officiis at, voluptatem similique tempora voluptate quia in ea
+                minima earum consequatur eveniet, praesentium illum error
+                delectus optio nobis eaque distinctio! Suscipit nostrum?
+              </p>
+            </div>
+            <div class="folder-button">
+              <button class="read-more">read more</button>
+            </div>
+          </div>
+          <div class="first-column">
+              <div class="folder">
+                <div class="folder-headline">
+                  <p class="folder-name">Just Another Blog Post</p>
+                </div>
+              </div>
+              <div class="folder-info">
+                <ul class="info-list">
+                  <li><img src="src/img/blog/blog-user.png"  alt=""/></li>
+                  <li><p>By Author</p></li>
```

Comment:
For what `<p>` here and at all your `<li>`?

Reply:
My bad,fixed